### DISCUSSION TITLE
Certificate verify failed

### DISCUSSION THREAD

**Discussion by fnakashima** [Zero6992/chatGPT-discord-bot#183] [Q&A]
After getting the latest code-set, got this error.
Do I have to do anything on my end for this?
![image](https://user-images.githubusercontent.com/8720157/218423128-c7cffc97-0789-4dff-aa66-168c40130638.png)

By the way, I'm on the free trial period in OpenAI.

**Comment by Zero6992** [comment]
CERTIFICATE_VERIFY_FAILED error might be caused by an outdated certifi package in Python
Here's a relevant discussion: https://stackoverflow.com/questions/51925384/unable-to-get-local-issuer-certificate-when-using-requests-in-python

**Reply by fnakashima** [reply]
Thanks for the info. I've tried several solutions from the link but none of them worked for me.
It's gonna be a long way...

**Reply by Zero6992** [reply]
Are you using a VPN?

**Reply by fnakashima** [reply]
No. Just running the programme in the local env on Windows 10.

**Comment by Cloner-X** [comment]
can you please help me with this certificate error  (.ssl.c:992)
![Screenshot 2023-02-16 at 4 55 49 AM](https://user-images.githubusercontent.com/90726887/219222791-dbbea8a4-af96-4d1f-b4da-ae3d2e5ca2eb.png)
